backend/src/plugins/Mutes/functions/clearExpiredMutes.ts

```diff
@@ -14,6 +15,13 @@ export async function clearExpiredMutes(pluginData: GuildPluginData<MutesPluginT
         if (muteRole) {
           await member.removeRole(muteRole);
         }
+        if (mute.roles_to_restore) {
+          const memberOptions: MemberOptions = {};
+          memberOptions.roles = Array.from(
+            new Set([...mute.roles_to_restore, ...member.roles.filter(x => x !== muteRole)]),
+          );
+          member.edit(memberOptions);
```

Comment:
Might need some validation here to make sure each role we try to restore actually exists

Reply:
Fixed with 499853b